packages/web-client/app/utils/events.ts

```diff
@@ -0,0 +1,20 @@
+export type UnbindEventListener = () => void;
+
+export interface Emitter {
+  on(event: string, cb: Function): UnbindEventListener; // eslint-disable-line no-unused-vars
+}
+
+export class DappEvents {
```

Comment:
@lukemelia I renamed this to avoid the accidental imports of node's event emitter.

Reply:
I'm a fan of avoiding the accidental imports but don't love the name. How about `SimpleEmitter` or `MinimalEmitter` instead?